examples/src/main/kotlin/com/samples/jariko/RpgSamples.kt

```diff
@@ -0,0 +1,70 @@
+package com.samples.jariko
+
+import com.smeup.rpgparser.execution.Configuration
+import com.smeup.rpgparser.execution.JarikoCallback
+import com.smeup.rpgparser.execution.executePgmWithStringArgs
+import com.smeup.rpgparser.execution.getProgram
+import com.smeup.rpgparser.interpreter.ISymbolTable
+import com.smeup.rpgparser.jvminterop.JavaSystemInterface
+import com.smeup.rpgparser.rpginterop.DirRpgProgramFinder
+import java.io.File
+
+/**
+ * This function execute fibonacciOf number passed as argument
+ * The rpgle program in retrieve from local sources from programPath
+ * @param fibonacciOf integer to calculate fibonacci series
+ */
+fun fibonacciOf(fibonacciOf: Int) {
+    val programPath = "examples/src/main/kotlin/com/samples/jariko"
+    val programName = "fibonacci.rpgle"
+    val programArgs = listOf(fibonacciOf.toString())
+    val jarikoCallback = JarikoCallback(
+            exitInRT = { false },
+            onExitPgm = { _: String, symbolTable: ISymbolTable, _: Throwable? ->
+                println(symbolTable["FINAL_VAL"].asInt().value)
```

Comment:
Printing of the calculated number is not what I would. I would like the function to return the calculated number